### PR TITLE
fix: Initialize ColorMapping with all values

### LINKED ISSUE #675: Can't change color mapping
**Describe the bug**
In some instances it's impossible to change color mapping of particular values.

**To Reproduce**
1. [Open Bathing water quality (with Hierarchie) dataset (INT data source)](https://test.visualize.admin.ch/en/browse/dataset/https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Fubd0104%2F11?previous=%7B%22includeDrafts%22%3Atrue%2C%22order%22%3A%22SCORE%22%2C%22search%22%3A%22bath%22%7D&dataSource=Int).
2. Start a visualization.
3. Change Horizontal Axis to _Parameter_.
4. Reorder _Date_ filter to be the top-most filter.
5. Filter _Date_ by 30.05.2007.
6. Choose _Observation Location_  as Color.
7. Select all filter values using Select all option.
8. See that color mapping only applies to "Clendy" – when doing further filtering, it's impossible to map colors to new values (which are blue by default).

**Expected behavior**
It should always be possible to change color mapping.

**Video**
https://user-images.githubusercontent.com/52032047/187922301-c1d50ed8-f331-48dd-b4e1-fe3235014d36.mov

**Environment (please complete the following information):**
 - TEST v3.7.10
 - Chrome 104, Safari 15.6

### PR DESCRIPTION
**Pull request by bprusinowski**
Fixes #675.

In some cases, the dimension used to create ColorMapping is filtered; in such situations users can't set color values for new values which appeared on chart during filtering. This PR fixes that by firing `handleRecomputeColorMapping` on first load of color mapping to assure it will use all dimension's values.

### CODE REVIEW

**Comment by ptbrowne** [comment]
LGTM, I think that'd be good to find a way to test this, would you see a way ?

File: app/configurator/components/filters.tsx [hunk 1]
```diff
@@ -354,10 +355,10 @@ const MultiFilterContent = ({
     return !!getColorMapping(config, colorConfigPath);
   }, [colorConfigPath, config]);
 
-  // Initialize color mapping with all values.
+  // Initialize color mapping with all values, not randomizing the order.
   useEffect(() => {
     if (hasColorMapping) {
-      handleRecomputeColorMapping();
+      handleRecomputeColorMapping(false);
```
[the remark this answers is not in the document]
Comment (bprusinowski) [reply app/configurator/components/filters.tsx]: Good point, done!

**Comment by bprusinowski** [comment]
Not at first glance, but I can try :)

**Comment by ptbrowne** [comment]
I do not see how to easily work out a test, I added an issue to tackle to add more E2E tests in 1 go : https://github.com/visualize-admin/visualization-tool/issues/692

**Pull request merged**